lib/index.js

```diff
@@ -0,0 +1,27 @@
+var _ = require('lodash');
```

Comment:
Seems a little odd that this file is index.js - this looks like a util file. I guess iamsetup module is really a util, so that makes sense.